File: content/cpp/templates/class.cpp
// Class Definition

#include <iostream>
#include <string>
using namespace std;

class Person {
private:
    string name;
    int age;

public:
    // Constructor
    Person(string n, int a) : name(n), age(a) {}
    
    // Methods
    string greet() {
        return "Hello, I'm " + name + "!";
    }
    
    string birthday() {
        age++;
        return "Happy birthday! Now " + to_string(age) + " years old.";
    }
    
    // Getters
    string getName() { return name; }
    int getAge() { return age; }
};

int main() {
    Person person("Alice", 25);
    
    cout << person.greet() << endl;
    cout << person.birthday() << endl;
    
    return 0;
}

File: content/python/templates/class.py
# Class Definition

class Person:
    """A simple Person class."""
    
    def __init__(self, name, age):
        self.name = name
        self.age = age
    
    def greet(self):
        return f"Hello, I'm {self.name}!"
    
    def birthday(self):
        self.age += 1
        return f"Happy birthday! Now {self.age} years old."

# Create an instance
person = Person("Alice", 25)
print(person.greet())
print(person.birthday())
